src/GitRepository.cc

```diff
@@ -103,12 +103,12 @@ void GitRepository::init(const std::filesystem::path& file_path)
 {
     repo_ = repository_open(repo_path_);
 
-    if (repo_.get() == nullptr)
+    if (not repo_)
```

Comment:
> if (git_repository_open(&repo, repo_path.c_str()))
>         return gul14::cat("repository_open: ", git_error_last()->message);

If repository_open fails, it returns a string with error message. How can "not repo_" be true if the string is never empty?

Reply:
Nevermind, I saw you overloaded the LiGitPointer constructor with a string. Looks good!

Reply:
Fixed explicit